contrib/openapi.yml

```diff
@@ -0,0 +1,170 @@
+swagger: "2.0"
+info:
+  title: Kostyor API
+  description: Aims & Launches Grenades
+  version: 1.0.0
+schemes:
+  - http
+consumes:
+  - application/json
+produces:
+  - application/json
+paths:
+  /environments:
```

Comment:
Ok, it's just a suggestion from my side. I don't know, maybe I understand things wrong, but to me cluster means homogenous machines and OpenStack setup consists of various roles (controllers with API, storages with disks, etc).